src/components/select/select.js

```diff
@@ -466,6 +461,7 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
           loadingAsync: attr.mdOnOpen ? scope.$eval(attr.mdOnOpen) || true : false
         }).finally(function() {
           selectScope.isOpen = false;
+          element.focus();
```

Comment:
I suspect this might interfere with the accessibility work that @rschmukler has recently done on this component. I'll ask him to take a look.